result.php

```diff
@@ -0,0 +1,184 @@
+<?php
+require 'vendor/autoload.php';
+use Dotenv\Dotenv;
+$dotenv = Dotenv::createMutable(__DIR__);
+$dotenv->load();
+$apiKey = $_ENV["apiKey"];
+
+// Declaring array for storing error messages in array format.
+$errorArray = [];
+$fullName = "";
+$trimmed = "";
+// Storing regex for checking.
+$namePattern = '/^[a-zA-Z]+$/';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['Submit'])) {
+  $fName = $_POST['fName'];
+  $lName = $_POST['lName'];
+  $fullName = $fName . " " . $lName;
+  $trimmed = preg_replace($namePattern, " ", $fullName);
```

Comment:
`$trimmed = preg_replace($namePattern, " ", $fullName);`
Explain what we are doing with this statement.